tests/external/test_stereoscope.py

```diff
@@ -0,0 +1,26 @@
+import scvi
```

Comment:
this needs to be a proper test, see the examples in test_models.py, just needs to be in function called `test_stereoscope()`.

You should also use the synthetic iid dataset for simplicity.